jenkins/internal-ci/generic/pr/cortx_deployment/cortx-rgw.groovy

```diff
@@ -168,7 +171,7 @@ pipeline {
                 script { build_stage = env.STAGE_NAME }
                 script {
                     try {
-                        def buildCortxAllImage = build job: 'cortx-docker-images-for-PR', wait: true,
+                        def buildCortxAllImage = build job: '/Cortx-PR-Build/Cortx-Deployment/Generic/cortx-docker-images-for-PR', wait: true,
```

Comment:
this path will change as well after successful test

Reply:
Done